src/walletConnect/screens/useIsDappListed.ts

```diff
@@ -0,0 +1,16 @@
+import { useSelector } from 'react-redux'
+import { dappsListSelector } from 'src/dapps/selectors'
+
+export const useIsDappListed = (dappName?: string, dappUrl?: string) => {
+  const dappsList = useSelector(dappsListSelector)
+
+  return (
+    !!dappName &&
+    !!dappUrl &&
+    !!dappsList.find(
+      (dapp) =>
+        dapp.name.toLowerCase() === dappName.toLowerCase() &&
```

Comment:
Checking the name too was a good idea, unfortunately, the listed name doesn't always match the `peerMeta` name reported.
For instance we have `Ubeswap` in our dapp list, but its `peerMeta.name` is `Ubeswap Interface`.

Reply:
i missed/didn't expect this...if this is the case then comparing the name doesn't make sense anymore, there might be a scenario where the listed name and peerMeta does not match at all? i removed the check for dapp name in this hook